pages/contact.js

```diff
@@ -10,15 +10,15 @@ export default function Contact() {
                     name="name"
                     className={styles.name}
                     id="inputName"
-                    oninput="checkInput()"
+                    onInput="checkInput()"
```

Comment:
The value of this `onInput` attribute is _calling_ `checkInput`. Instead, it should simply reference the function name without the parens. For example:

```
onInput="checkInput"
```

Reply:
Also, instead of rolling your own validator, use https://react-hook-form.com/

Reply:
thanks for the demonstration! check